docs/update-cli.md

```diff
@@ -0,0 +1,26 @@
+# Update OpenAPI CLI
+
+If you've already installed OpenAPI CLI, you'll want to keep it as up to date as possible. Staying on top of versions ensures that you get the latest functionality and bug fixes. Depending on how you originally installed OpenAPI CLI, there are two ways to update it.
+
+:::success Tip
+Before you begin, ensure that you have the latest version of npm installed.
+:::
+
+## How to update your openapi-starter installation
+
+If you installed OpenAPI CLI using the [openapi-starter template](https://github.com/Redocly/openapi-starter) follow these steps:
+
+1. Navigate to and open `package.json` in any text editor.
+2. Find the line that contains `"@redocly/openapi-cli":` followed by the version number.
+3. Change the version number to the latest one in the [Changelog](https://redocly.com/docs/cli/changelog/).
+4. Save the changes then close `package.json`.
```

Comment:
I think they'll have to run `npm install` after step 4, right?

Reply:
Yes, and one more step to commit the `package-lock.json` to the git repository.

Reply:
So to confirm:
```suggestion
4. Save the changes then close `package.json`.
5. Run `npm install` from the project's root directory on the command line.
6. Commit the changed `package.json` and `package-lock.json` files to your Git repository.
```